wormhole-connect/src/components/InputContainer.tsx

```diff
@@ -6,7 +6,7 @@ type StyleProps = { bg?: string };
 const useStyles = makeStyles<StyleProps>()((theme: any, { bg }) => ({
   input: {
     width: '100%',
-    padding: '16px',
+    padding: theme.spacing(1.5),
```

Comment:
How'd you decide between when to set specific values vs when to use `theme.spacing()`?

Reply:
I used the theme spacing for gap/padding/margin anywhere it was available.

Reply:
Thinking down the line - might it be tricky for someone to figure out what's controlled via `theme.spacing()` vs `px`. The latter is annoying to track, but I'm thinking about someone who changes theme.spacing(), sees that it doesn't change everything they want, and needing to go through all the `px` ones anyways.

If you think there's a way to get to relying on `theme` everywhere down the line then this makes sense.

Reply:
Sorry, should have added that to the description - I think we should consider moving all the spacing to `theme.spacing` or the `sx` prop so that we can support "comfortable" vs "compact" with 1 setting. all of these being explicitly set like this would never have been properly customizable by integrators.